src/app/EventManagement.cpp

```diff
@@ -25,13 +24,16 @@
 #include <support/CodeUtils.h>
 #include <support/ErrorStr.h>
 #include <support/logging/CHIPLogging.h>
+#include <system/SystemMutex.h>
 #include <system/SystemTimer.h>
 
 using namespace chip::TLV;
 
 namespace chip {
 namespace app {
 static EventManagement sInstance;
+static System::Mutex sEventLock;
```

Comment:
Why is this not a member of EventManagement?  Might be worth documenting that, at least....

Reply:
forgot to move it, just update it as member variable inside EventManagement.